dev/bench/dune

```diff
@@ -15,5 +15,7 @@
 
 (executable
  (name timelog2html)
+ (public_name coqtimelog2html)
```

Comment:
Shouldn't we have a more coq-y name to be sure this will not conflict with another tool?